Makefile

```diff
@@ -104,9 +104,24 @@ verifyversion: build
 test: build verifyversion
 	go test -race ./...
 
+# List of files we don't need to track coverage for.
+# (Include a reason for each.)
+#
+# lex.go, y.go: Generated by Ragel and goyacc. Many cases cannot be exercised.
+# mock_protocol.go: Generated by gomock, which is thoroughly tested.
+COVER_IGNORE_FILES = \
+	 idl/internal/lex.go \
+	 idl/internal/y.go \
+	 thrifttest/mock_protocol.go
+
+# literal space
+space :=
+space +=
+
 .PHONY: cover
 cover:
-	go test -v -covermode=atomic -coverprofile cover.out -coverpkg=./... ./...
+	go test -v -covermode=atomic -coverprofile cover.full.out -coverpkg=./... ./...
+	grep -v "$(subst $(space),\|,$(COVER_IGNORE_FILES))" cover.full.out > cover.out
```

Comment:
why do we need to replace spaces with `|`? nicer output?

Reply:
We're building a regular expression so we need to replace the spaces with `|` clauses.

Reply:
whoops, gotcha